web/inc/main.php

```diff
@@ -175,6 +175,12 @@ function render_page($user, $TAB, $page) {
 	$__template_dir = dirname(__DIR__) . "/templates/";
 	$__pages_js_dir = dirname(__DIR__) . "/js/pages/";
 
+	// Panel
+	$panel = top_panel(empty($_SESSION["look"]) ? $_SESSION["user"] : $_SESSION["look"], $TAB);
```

Comment:
Leave this below the header include otherwise you can strange behaviour in page loading

Reply:
See Discord

Reply:
ah, I see, will do